test/product-import.spec.coffee

```diff
@@ -308,9 +308,9 @@ describe 'ProductImport unit tests', ->
         done()
       .catch done
 
-  xdescribe '::_generateSlug', ->
+  describe '::_generateSlug', ->
 
-    it 'should generate valid slug', ->
+    xit 'should generate valid slug', ->
```

Comment:
re enabled the test->  https://github.com/sphereio/sphere-product-import/commit/6f4b4969d543efd0d763e239fb71663e1c52646c